metron-analytics/metron-profiler/README.md

```diff
@@ -277,25 +282,103 @@ The values can be changed on disk and then the Profiler topology must be restart
-| profiler.period.duration              | The duration of each profile period.  This value should be defined along with `profiler.period.duration.units`.
-| profiler.period.duration.units        | The units used to specify the `profiler.period.duration`.
-| profiler.ttl                          | If a message has not been applied to a Profile in this period of time, the Profile will be forgotten and its resources will be cleaned up. This value should be defined along with `profiler.ttl.units`.
-| profiler.ttl.units                    | The units used to specify the `profiler.ttl`.
-| profiler.hbase.salt.divisor           | A salt is prepended to the row key to help prevent hotspotting.  This constant is used to generate the salt.  Ideally, this constant should be roughly equal to the number of nodes in the Hbase cluster.
-| profiler.hbase.table                  | The name of the HBase table that profiles are written to.
-| profiler.hbase.column.family          | The column family used to store profiles.
-| profiler.hbase.batch                  | The number of puts that are written in a single batch.
-| profiler.hbase.flush.interval.seconds | The maximum number of seconds between batch writes to HBase.
-
-After altering the configuration, start the Profiler.
+| [`profiler.input.topic`](#profilerinputtopic) | The name of the Kafka topic from which to consume data.
+| [`profiler.output.topic`](#profileroutputtopic) | The name of the Kafka topic to which profile data is written.  Only used with profiles that define the [`triage` result field](#result).
+| [`profiler.period.duration`](#profilerperiodduration) | The duration of each profile period.  
+| [`profiler.period.duration.units`](#profilerperioddurationunits) | The units used to specify the [`profiler.period.duration`](#profilerperiodduration).  
+| [`profiler.workers`](#profilerworkers) | The number of worker processes for the topology.
+| [`profiler.executors`](#profilerexecutors) | The number of executors to spawn per component.
+| [`profiler.ttl`](#profilerttl) | If a message has not been applied to a Profile in this period of time, the Profile will be forgotten and its resources will be cleaned up. 
+| [`profiler.ttl.units`](#profilerttlunits) | The units used to specify the `profiler.ttl`.
+| [`profiler.hbase.salt.divisor`](#profilerhbasesaltdivisor) | A salt is prepended to the row key to help prevent hotspotting. 
+| [`profiler.hbase.table`](#profilerhbasetable) | The name of the HBase table that profiles are written to.
+| [`profiler.hbase.column.family`](#profilerhbasecolumnfamily) | The column family used to store profiles.
+| [`profiler.hbase.batch`](#profilerhbasebatch) | The number of puts that are written to HBase in a single batch.
+| [`profiler.hbase.flush.interval.seconds`](#profilerhbaseflushintervalseconds) | The maximum number of seconds between batch writes to HBase.
```

Comment:
Is there a reason to not use fixed width columns?  I know this is not a very complicated table, but in the past [we have had issues](https://github.com/apache/metron/commit/f6c253f44fe7ca5a6b31c46075f537940751c934#diff-dba284384333126318019f762880ace4) where a table formatting mistake was causing information to fail to display and was hard to identify when looking only at the raw markdown.  No strong opinions either way.